smdebug/core/collection.py

```diff
@@ -45,6 +45,10 @@ class CollectionKeys:
     FULL_SHAP = "full_shap"
     TREES = "trees"
 
+    @classmethod
+    def values(cls):
+        return [value for name, value in vars(cls).items() if name.isupper()]
```

Comment:
Also, can you add a test for this values method. What else can vars() return

Reply:
Removed.